.ado/pipelines/templates/stages-loadtest-azure.yaml

```diff
@@ -66,22 +71,35 @@ stages:
       - script: |
           echo "*** Setting loadTestTargetFQDN to $(frontdoor_fqdn)"
           echo "##vso[task.setvariable variable=azureLoadTestTargetFQDN]$(frontdoor_fqdn)"
-        displayName: 'Set Loadtest targetUrl to Azure Front Door'
 
-    # Set loadTestTargetFQDN to the FQDN specified in the pipeline parameters
+          echo "*** Setting loadTestBaseline to $(azureLoadTestBaseline)"
+          echo "##vso[task.setvariable variable=loadTestBaseline]$(azureLoadTestBaseline)"
+
+        displayName: 'Set Azure Load Test variables'
```

Comment:
I'm not a big fan of this condensed task syntax - especially when display name now comes at the bottom. can you maybe rewrite to the full task?

Reply:
I don't have a strong preference and do not insist on keeping it like that. Will change it to the more prescriptive task syntax.